.gitignore

```diff
@@ -3,6 +3,9 @@
 bin/
 /tf-operator
 vendor/
+testbin/*
```

Comment:
Where is this being generated from?

Reply:
It's from `make test` command. https://github.com/kubeflow/tf-operator/blob/91ae0a8219085230d571f1c1df0808ca9b20d09e/Makefile#L52-L56

command automatically creates `./testbin`